storage/statedb/database.go

```diff
@@ -357,6 +369,11 @@ func (db *Database) DiskDB() database.DBManager {
 	return db.diskDB
 }
 
+// TrieNodeCache retrieves the trieNodeCachetheof the trie database.
```

Comment:
`trieNodeCachetheof` -> `trieNodeCache of`

Reply:
Good. I reflect. :)